src/fluree/db/query/fql/syntax.cljc

```diff
@@ -228,3 +239,12 @@
                     {:status  400
                      :error   :db/invalid-query
                      :reasons (m/explain ::analytical-query qry {:registry registry})}))))
+
+(defn coerce
+  [qry]
+  (try* (query-coercer qry)
+        (catch* _e
+                (throw (ex-info "Invalid Query"
+                                {:status  400
+                                 :error   :db/invalid-query
+                                 :reasons (m/explain ::query qry {:registry registry})})))))
```

Comment:
You might want to try wrapping this in `(me/humanize ...)` but that's also out of scope for this PR, so up to you.